pkg/logs/launchers/integration/launcher.go

```diff
@@ -156,58 +268,109 @@ func (s *Launcher) makeFileSource(source *sources.LogSource, logFilePath string)
 	return fileSource
 }
 
-// TODO Change file naming to reflect ID once logs from go interfaces gets merged.
 // createFile creates a file for the logsource
-func (s *Launcher) createFile(id string) (string, error) {
-	logFilePath := s.integrationLogFilePath(id)
+func (s *Launcher) createFile(source string) (*FileInfo, error) {
+	filepath := s.integrationLogFilePath(source)
 
-	file, err := os.Create(logFilePath)
+	file, err := os.Create(filepath)
 	if err != nil {
-		return "", nil
+		return nil, err
+	}
+	ddLog.Info("Successfully created integrations log file.")
+
+	err = file.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	fileInfo := &FileInfo{
+		filename:     filepath,
+		lastModified: time.Now(),
+		size:         0,
 	}
-	defer file.Close()
 
-	return logFilePath, nil
+	return fileInfo, nil
 }
 
 // integrationLoglogFilePath returns a file path to use for an integration log file
 func (s *Launcher) integrationLogFilePath(id string) string {
-	fileName := strings.ReplaceAll(id, " ", "-")
-	fileName = strings.ReplaceAll(fileName, ":", "_") + ".log"
+	fileName := strings.ReplaceAll(id, ":", "_") + ".log"
 	logFilePath := filepath.Join(s.runPath, fileName)
 
 	return logFilePath
 }
 
-// ensureFileSize enforces the max file size for files integrations logs
-// files. Files over the set size will be deleted and remade.
-func (s *Launcher) ensureFileSize(logFilePath string) error {
-	maxFileSizeSetting := pkgconfigsetup.Datadog().GetInt64("logs_config.integrations_logs_files_max_size")
-	maxFileSizeBytes := maxFileSizeSetting * 1024 * 1024
-
-	fi, err := os.Stat(logFilePath)
+// makeFile deletes log files and creates a new empty file with the
+// same name
+func (s *Launcher) makeFile(filepath string) error {
```

Comment:
The logic of this function is duplicated from lines 275-281. Can you re-use this function there?

Reply:
After some thought, I've decided to remove the `makeFile` function since it doesn't make much sense to wrap a simple function and do nothing else; after error checking, this function didn't end up saving many lines of code anyways.